### PR TITLE
Handle backend exceptions per HTTP code

### PR DESCRIPTION
**Pull request by luisferreira32**
The objective of this PR is that exceptions are handled per HTTP code, and the exception itself holds enough information to inform the front-end of its procedures.

### CODE REVIEW

File: .github/workflows/services-pr-checker.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ on:
       - edited
       - reopened
     branches:
-      - "services/**"
+      - "master"
```
Comment (luisferreira32) [inline .github/workflows/services-pr-checker.yml:9]: hotfix 😄

File: services/fitter/server.py [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import os
 from service import service
 from http.server import HTTPServer
-from intmodules import config_logger, logger, MPBRequestHandler
-from binascii import Error as ReadingBase64Error
-from intmodules import DuplicatedEntryError, NanValueFoundError
+from intmodules import MPBRequestHandler # config_logger, logger,
```
Comment (luisferreira32) [inline services/fitter/server.py:4]: Comments to satisfy the linter 🛩️

File: services/fitter/server.py [hunk 3]
```diff
@@ -16,20 +15,8 @@ def do_POST(self):
...
-            self._return_400(response)
-        except NanValueFoundError:
-            response = "Bad Request: NanValueFoundError"
-            self._return_400(response)
-        except DuplicatedEntryError:
-            response = "Bad Request: DuplicatedEntryError"
-            self._return_400(response)
-        except ValueError:
-            response = "Bad Request: ValueError"
+        except (InvalidRequestError, ValidationError, ValueError) as e:
```
Comment (MrToino) [inline services/fitter/server.py:18]: Just wondering if the parenthesis are redundant.

**Review by MrToino — APPROVED** [review]
This new implementation is much simpler and effective. However we shall do extensive tests to document what are the most common errors that we can expect.

File: services/fitter/server.py (hunk 3 above)
Comment (luisferreira32) [reply services/fitter/server.py:18]: Unfortunately, they are required.

File: .github/workflows/services-pr-checker.yml (hunk 1 above)
Comment (luisferreira32) [reply .github/workflows/services-pr-checker.yml:9]: Side note, this is not really an issue for user experience / api consumers, it's just our development flow, it wouldn't make sense to open an issue.

**Pull request merged**